### ISSUE TITLE
Unable to build on MacOS M2

### ISSUE THREAD

**Issue by stanle-cs** [pr2502/ra-multiplex#70]
Hi, I tried to build the project on a MacBook M2 and the --release build failed. Interestingly enough, the regular `cargo build` works. I'm not sure what the problem is.

How to reproduce:
1. `git clone ...` the project
2. `cargo build --release`

Output:
```
   Compiling proc-macro2 v1.0.66
   Compiling libc v0.2.154
   Compiling unicode-ident v1.0.11
   Compiling once_cell v1.18.0
   Compiling autocfg v1.1.0
   Compiling cfg-if v1.0.0
   Compiling parking_lot_core v0.9.8
   Compiling serde v1.0.186
   Compiling lazy_static v1.4.0
   Compiling utf8parse v0.2.1
   Compiling smallvec v1.11.0
   Compiling anstyle-parse v0.2.1
error: failed to run custom build command for `parking_lot_core v0.9.8`

Caused by:
  process didn't exit successfully: `[redacted]/ra-multiplex/target/release/build/parking_lot_core-0b7f71150f9e05ed/build-script-build` (signal: 9, SIGKILL: kill)
warning: build failed, waiting for other jobs to finish...
```

**Comment by pr2502** [comment]
I can't find any relevant issue in the parking_lot crate, but our version (0.9.8) is over a year old and can be upgraded. Can you try if `cargo update -p parking_lot_core` helps?

**Comment by simnalamburt** [comment]
Could not reproduce this issue in latest ra-multiplex (0.2.5). I'd like to recommend closing the issue.

**Comment by pr2502** [comment]
Thanks for the info. I don't have a way to test on a mac so I'm closing the issue. Feel free to reopen or create a new issue if there is a problem again.

**Comment by simnalamburt** [comment]
Of course. Everything is working fine now, and I’ve even set up automated builds and Homebrew packaging through CI. If any issues arise, I’ll be immediately notified via CI.

###### Reference
- https://github.com/simnalamburt/homebrew-x/#ra-multiplex
